### PR TITLE
13.0 Change prices

### PR DESCRIPTION
**Pull request by em230418**

### CODE REVIEW

File: reminder_crm_next_action_time/__manifest__.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "support": "apps@it-projects.info",
     "website": "https://twitter.com/gabbasov_dinar",
     "license": "GPL-3",
-    "price": 21.00,
+    "price": 10.50,
```
Comment (Ramil-Mukhametzyanov) [inline reminder_crm_next_action_time/__manifest__.py]: price = 11.0

File: project_task_search_custom/__manifest__.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
     'license': 'LGPL-3',
     'website': "https://it-projects.info",
     'version': '1.1.0',
-    'price': 15.00,
+    'price': 7.50,
```
Comment (Ramil-Mukhametzyanov) [inline project_task_search_custom/__manifest__.py]: price = 9.0 (min value)

**Review by Ramil-Mukhametzyanov — COMMENTED** [review]
What about the project_task_order_kanban_state module?

File: reminder_issue_deadline/__manifest__.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
     'license': 'LGPL-3',
     'category': 'Reminders and Agenda',
     'website': 'https://www.matmoz.si',
-    'price': 21.00,
+    'price': 10.50,
```
Comment (Ramil-Mukhametzyanov) [inline reminder_issue_deadline/__manifest__.py]: price = 11.0

**Review by Ramil-Mukhametzyanov — COMMENTED** [review]
What about the reminder_hr_recruitment module?

**Comment by Ramil-Mukhametzyanov** [comment]
> What about the reminder_hr_recruitment module?

price = 11.0

File: web_multi_attachment_base/__manifest__.py [hunk 4]
```diff
@@ -14,7 +14,7 @@
     "support": "apps@it-projects.info",
     "website": "https://it-projects.info/team/GabbasovDinar",
     "license": "LGPL-3",
-    "price": 19.00,
+    "price": 9.50,
```
Comment (Ramil-Mukhametzyanov) [inline web_multi_attachment_base/__manifest__.py]: price = 10.0

**Comment by Ramil-Mukhametzyanov** [comment]
> What about the reminder_hr_recruitment module?

Ok. It is deleted from the branch

**Comment by Ramil-Mukhametzyanov** [comment]
> What about the reminder_hr_recruitment module?

It is also deleted

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Comment by ilmir-k** [comment]
I approve to merge it now

**Pull request merged**